src/node_dotenv.cc

```diff
@@ -277,12 +229,9 @@ Dotenv::ParseResult Dotenv::ParsePath(const std::string_view path) {
   return ParseResult::Valid;
 }
 
-void Dotenv::AssignNodeOptionsIfAvailable(std::string* node_options) const {
```

Comment:
This removed implementation doesn't make any unnecessary copy, but your new implementation does.

Reply:
It's not unneccessery. It needs to be an `std::string` for `ProcessGlobalArgsInternal`